kura/test/org.eclipse.kura.nm.test/src/test/java/org/eclipse/kura/nm/configuration/NMSettingsConverterTest.java

```diff
@@ -51,6 +51,9 @@ public class NMSettingsConverterTest {
     Boolean hasAnIllegalArgumentExceptionThrown = false;
     Boolean hasAGenericExecptionBeenThrown = false;
 
+    private static List<List<Byte>> expectedDnsList = Arrays
+            .asList(Arrays.asList(new Byte[] { 32, 1, 72, 96, 72, 96, 0, 0, 0, 0, 0, 0, 0, 0, -120, 68 }));
+
```

Comment:
It is a constant therefore you'll need the `final` keyword. Furthermore you'll need to write it as all capital.

Finally I prefer to pass the expectation method a list of constants for readability. This means that this becomes:

 ```suggestion
    private static fina List<Byte> IP6_BYTE_ARRAY_ADDRESS = Arrays.asList(new Byte[] { 32, 1, 72, 96, 72, 96, 0, 0, 0, 0, 0, 0, 0, 0, -120, 68 });

```